.github/workflows/test_destinations.yml

```diff
@@ -23,6 +23,8 @@ env:
   RUNTIME__SENTRY_DSN: https://6f6f7b6f8e0f458a89be4187603b55fe@o1061158.ingest.sentry.io/4504819859914752
   RUNTIME__LOG_LEVEL: ERROR
   RUNTIME__DLTHUB_TELEMETRY_SEGMENT_WRITE_KEY: TLJiyRkGVZGCi2TtjClamXpFcxAA1rSB
+  ALL_DESTINATIONS: "[\"duckdb\", \"postgres\"]"
```

Comment:
the name is misleading :/
here also test redshift and filesystem (including s3 and gs buckets) so we should include it here, maybe also change the name to include redshift and filesystem

Reply:
we ignore certain paths so tests do not fail on dependencies